src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+  </head>
+  <body class="page__content">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        class="card__image card__image--position"
+        alt="Samsung Galaxy S22 2022 8/128GB Green"
+      >
+
+      <h2 class="card__title card__title--position">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code card__code--position">
+        Product code: 195434
+      </p>
+
+      <div class="reviews reviews__stars--4 reviews__stars--position">
+        <span class="stars-star"></span>
+        <span class="stars-star"></span>
+        <span class="stars-star"></span>
+        <span class="stars-star"></span>
+        <span class="stars-star"></span>
+        <span class="reviews__card"> Reviews: 5</span>
+      </div>
+
+      <p class="price price--position">
```

Comment:
![image](https://user-images.githubusercontent.com/54936044/233382065-a2992ec5-26c2-4cfb-b259-feb0610d015c.png)
Please change to card__price, because price block element is the child of card element